packages/fx-core/src/core/middleware/projectMigratorV3.ts

```diff
@@ -337,13 +338,16 @@ export async function configsMigration(context: MigrationContext): Promise<void>
           );
           if (obj["manifest"]) {
             const bicepContent = readBicepContent(context);
-            // convert every name
-            const envData = jsonObjectNamesConvertV3(
-              obj["manifest"],
-              "manifest.",
-              FileType.CONFIG,
-              bicepContent
-            );
+            const teamsfx_env = fs
```

Comment:
can we add a step to create new .env files with TEAMSFX_ENV variable? so we don't need to duplicate this logic multiple times

Reply:
sure

Reply:
I want to add this logic once at the beginning of config migration, as config migration will be the first migration. what do u think? @blackchoey